### PR TITLE
RUN-16065 Revamp Cluster Installation Troubleshooting

### PR DESCRIPTION
**Pull request by gshaibi**
- **Update cluster installation instructions and troubleshooting**
- **.**

### CODE REVIEW

**Review by roir — APPROVED** [review]

File: docs/admin/runai-setup/cluster-setup/cluster-prerequisites.md [hunk 1]
```diff
@@ -133,6 +133,7 @@ Follow the [Getting Started guide](https://docs.nvidia.com/datacenter/cloud-nati
     * NVIDIA drivers may already be installed on the nodes. In such cases, use the NVIDIA GPU Operator flags `--set driver.enabled=false`. [DGX OS](https://docs.nvidia.com/dgx/index.html){target=_blank} is one such example as it comes bundled with NVIDIA Drivers.
     <!-- * To work with *containerd* (e.g. for Tanzu), use the [defaultRuntime](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/getting-started.html#chart-customization-options){target=_blank} flag accordingly. -->
     * To use [Dynamic MIG](../../../Researcher/scheduling/fractions.md#dynamic-mig), the GPU Operator must be installed with the flag `mig.strategy=mixed`. If the GPU Operator is already installed, edit the clusterPolicy by running ```kubectl patch clusterPolicy cluster-policy -n gpu-operator --type=merge -p '{"spec":{"mig":{"strategy": "mixed"}}}```
```
[the remark this answers is not in the document]
Comment (ozRunAI) [reply docs/admin/runai-setup/cluster-setup/cluster-prerequisites.md:135]: I think its relevant

File: docs/admin/troubleshooting/cluster-health-check.md [hunk 2]
```diff
@@ -90,6 +91,16 @@ When a cluster's status shows *Service issues*, this means that one or more Run:
 
 3. If the issue persists, contact Run:ai support for assistance.
 
+## Cluster has missing prerequisites
+
+When a cluster's status displays Missing prerequisites, it indicates that at least one of the [Mandatory Prerequisites](../runai-setup/cluster-setup/cluster-prerequisites.md#prerequisites-in-a-nutshell) has not been fulfilled. In such cases, Run:ai services may not function properly.
+
+If you have ensured that all prerequisites are installed and the status still shows Missing prerequisites, follow these steps:
+
+1. Check the message in the UI for further details regarding the missing prerequisites.
```
Comment (ozRunAI) [inline docs/admin/troubleshooting/cluster-health-check.md:100]: do not use the word UI

File: docs/admin/troubleshooting/cluster-health-check.md [hunk 3]
```diff
@@ -161,3 +172,26 @@ Submitting a Job allows you to verify that the Run:ai scheduling service is runn
 
 Log into the Run:ai user interface, and verify that you have a `Researcher` or `Research Manager` role.
 Go to the `Jobs` area. On the top right, press the button to create a Job. Once the form opens, you can submit a Job.
+
+## Advanced Troubleshooting
```
Comment (ozRunAI) [inline docs/admin/troubleshooting/cluster-health-check.md:176]:
```suggestion
## Advanced troubleshooting
```

File: docs/admin/troubleshooting/cluster-health-check.md [hunk 4]
```diff
@@ -161,3 +172,26 @@ Submitting a Job allows you to verify that the Run:ai scheduling service is runn
...
+
+### Run:ai public ConfigMap
+
+Run:ai services use the `runai-public` ConfigMap to store information about the cluster status. This ConfigMap can be helpful in troubleshooting issues with Run:ai services.
+Inspect the ConfigMap by running:
+
+```bash
+kubectl get cm runai-public -oyaml | yq .data.runai-public
```
Comment (ozRunAI) [inline docs/admin/troubleshooting/cluster-health-check.md:184]: should we use yq also here? is it necessary in this example?

File: docs/admin/troubleshooting/cluster-health-check.md [hunk 5]
```diff
@@ -161,3 +172,26 @@ Submitting a Job allows you to verify that the Run:ai scheduling service is runn
...
+```
+
+### Resources not deployed / System Unavailable / Reconciliation Failed
```
Comment (ozRunAI) [inline docs/admin/troubleshooting/cluster-health-check.md:187]:
```suggestion
### Resources not deployed / System unavailable / Reconciliation failed
```

**Review by ozRunAI — CHANGES_REQUESTED** [review]

**Review by ozRunAI — APPROVED** [review]

File: docs/admin/runai-setup/cluster-setup/cluster-prerequisites.md [hunk 6]
```diff
@@ -216,6 +217,9 @@ helm install prometheus prometheus-community/kube-prometheus-stack \
 
 1. The Grafana component is not required for Run:ai.
 
+!!! Notes
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/cluster-setup/cluster-prerequisites.md:220]:
```suggestion
!!! Note
```

**Pull request merged**